### ISSUE TITLE
The check boxes and radio buttons don't work when I run an applet using cj3loader, but they work using the extension

### ISSUE THREAD

**Issue by mobluse** [leaningtech/cheerpj-meta#196]
The check boxes and radio buttons don't work fully when I run the applet using https://cjrtnc.leaningtech.com/3.0/cj3loader.js, but they work when I run it using the CheerpJ Applet Runner 10.6 from Chrome Web Store, see the applet Aritm on:
https://mobluse.github.io/aritm/
The error is that the boxes and buttons are not filled in and that they only work programwise (but not visually) as they should the first time they are clicked.

Source: https://github.com/mobluse/aritm

I've tested this with Chromium and Firefox in most updated Debian Bookworm on Raspberry Pi 4 B. The extension, of course, only works in Chromium. I load the extension instead of cj3loader by reloading the page and quickly click the orange extension button.

This is an educationally important applet and it would be good if it could work without needing to install an extension.

Maybe the cj3loader and extension are different versions.

**Comment by alexp-sssup** [comment]
Thanks for reporting this. The Applet Runner extension is still based on the old CheerpJ 2 architecture, although it is going to be upgraded soon. Some AWT logic must have been lost during the re-architecturing to CheerpJ 3. We'll take a look at soon as possible.

**Comment by mobluse** [comment]
I have the same app on six pages (three languages and without or with sound). I downgraded to CheerpJ 2.3 for one language: Swedish. Then the boxes and buttons work there fully. I also discovered that there is some support for sound (sun.audio) in 2.3 which doesn't seem to exist in 3.0. Some sound can be heard if you type e.g. letters instead of numbers. Swedish Aritm with and without sound:
https://mobluse.github.io/aritm/sun.audio/index-sv.html
https://mobluse.github.io/aritm/index-sv.html
Also, 2.3 seems to load faster.

English Aritm still has 3.0:
https://mobluse.github.io/aritm/sun.audio/index.html
https://mobluse.github.io/aritm/index.html
